docs/application/web/api/7.0/device_api/mobile/index.xml

```diff
@@ -1,233 +1,105 @@
+						<topic href="7.0/device_api/mobile/tizen/keymanager.html" label="Keymanager"/>
+						<topic href="7.0/device_api/mobile/tizen/ppm.html" label="PrivacyPrivilege"/>
+					</topic>
+					<topic href="7.0/device_api/mobile/index.html#Social" label="Social">
+						<topic href="7.0/device_api/mobile/tizen/calendar.html" label="Calendar"/>
+						<topic href="7.0/device_api/mobile/tizen/callhistory.html" label="Call History"/>
+						<topic href="7.0/device_api/mobile/tizen/contact.html" label="Contact"/>
+						<topic href="7.0/device_api/mobile/tizen/datasync.html" label="Data Synchronization"/>
+					</topic>
+					<topic href="7.0/device_api/mobile/index.html#System" label="System">
+						<topic href="7.0/device_api/mobile/tizen/feedback.html" label="Feedback"/>
+						<topic href="7.0/device_api/mobile/tizen/humanactivitymonitor.html" label="Human Activity Monitor"/>
+						<topic href="7.0/device_api/mobile/tizen/mediakey.html" label="Media Key"/>
+						<topic href="7.0/device_api/mobile/tizen/power.html" label="Power"/>
+						<topic href="7.0/device_api/mobile/tizen/sensor.html" label="Sensor"/>
+						<topic href="7.0/device_api/mobile/tizen/systeminfo.html" label="System Information"/>
+						<topic href="7.0/device_api/mobile/tizen/systemsetting.html" label="System Setting"/>
+						<topic href="7.0/device_api/mobile/tizen/time.html" label="Time"/>
+						<topic href="7.0/device_api/mobile/tizen/websetting.html" label="Web Setting"/>
+					</topic>
+					<topic href="7.0/device_api/mobile/index.html#UIX" label="UIX">
+						<topic href="7.0/device_api/mobile/tizen/voicecontrol.html" label="Voice Control"/>
+					</topic>
+					<topic href="7.0/device_api/mobile/index.html#Cordova" label="Cordova">
+						<topic href="7.0/device_api/mobile/tizen/cordova/console.html" label="Console"/>
+						<topic href="7.0/device_api/mobile/tizen/cordova/cordova.html" label="Cordova"/>
+						<topic href="7.0/device_api/mobile/tizen/cordova/device.html" label="Device"/>
+						<topic href="7.0/device_api/mobile/tizen/cordova/device-motion.html" label="Device Motion"/>
+						<topic href="7.0/device_api/mobile/tizen/cordova/dialogs.html" label="Dialogs"/>
+						<topic href="7.0/device_api/mobile/tizen/cordova/events.html" label="Events"/>
+						<topic href="7.0/device_api/mobile/tizen/cordova/file.html" label="File"/>
+						<topic href="7.0/device_api/mobile/tizen/cordova/filetransfer.html" label="File Transfer"/>
+						<topic href="7.0/device_api/mobile/tizen/cordova/globalization.html" label="Globalization"/>
+						<topic href="7.0/device_api/mobile/tizen/cordova/media.html" label="Media"/>
+						<topic href="7.0/device_api/mobile/tizen/cordova/networkInformation.html" label="Network Information"/>
+					</topic>
+					<topic href="7.0/device_api/mobile/index.html#Deprecated API" label="Deprecated API">
+						<topic href="7.0/device_api/mobile/tizen/libteec.html" label="LibTeec" deprecatedSince="6.5"/>
 					</topic>
```

Comment:
Question to this update.

With this update, only device Web API list is left, 
and other Web API lists such as ui_fw_api or w3c_api are removed.

As far as I remember, when index.xml is generated, 
the other Web API list is attached automatically to index.xml.

Does leaving only device Web API mean we will have only device API in Tizen 7.0 Web?

Reply:
I have just noticed that as well. The files have been generated automatically. I have to check that.

Reply:
@mijin85cho Thank you for noticing that, I have added the missing links.

Reply:
> @mijin85cho Thank you for noticing that, I have added the missing links.

Thanks for updating it!

Beside, about generating `index.xml`, I'd like to suggest one thing.

As I commented above, we are considering not handling  `toc.xml` in the future.
(Now we have many platform versions, so the `toc.xml` file is getting heavier and heavier.)
To separate the menu tree structure file in pieces, we think `index.xml` of each version/profile is a good alternative.

However as far as I know, new version `index.xml` file generation is also based on `toc.xml`.
so, would you please check if you can use previous `index.xml` file as a base when you generate a new `index.xml` file?
If my memory is wrong so you don't use `toc.xml` as the source to make `index.xml`, then you'd not need to update it.

Reply:
@mijin85cho @m-bialek,
Have all the issues been fixed on `index.xml`?
I can approve the PR if all the changes are completed.

Reply:
Apologies for the late response. Today or tomorrow I will check if I can generate a new `index.xml` based on the old one and update the file as @mijin85cho asked.

Reply:
@mijin85cho It seems that I would have to change some code in a generator tool in order to use the old `index.xml` as the base for a new one. It should be possible though. I do not remember using `toc.xml` as the source for the index file so I guess I do not have to update anything. However I generated `index.xml` once again and it seems to be the same (except additional sections like ui_fw_api or w3c_api which have to be added manually).

Reply:
Thanks for checking @m-bialek 
Other Web API sections such as ui_fw_api or w3c_api needs manual update to index.xml when these sections are changed. So If these sections copies previous version of themselves when generating index.xml, it would be OK.
In the future if we encounter any issue I'll ask you again to check, but for now, current looks enough.